cmd/otelopscol/main.go

```diff
@@ -54,7 +55,7 @@ func runInteractive(params service.CollectorSettings) error {
 		return fmt.Errorf("failed to construct the application: %w", err)
 	}
 
-	err = app.Run()
+	err = app.Run(context.Background())
```

Comment:
The upstream has inserted a wrapper method [here](https://github.com/open-telemetry/opentelemetry-collector/blob/main/cmd/otelcol/main.go#L46:L47), which may be worth including in this update.

Reply:
That code you linked to is not in 0.36 yet, unfortunately.

Reply:
[Here](https://github.com/open-telemetry/opentelemetry-collector/blob/v0.36.0/cmd/otelcol/main.go) is the v0.36.0 tagged version of it. Slightly different usage, but the `service` package appears to be present.